Problem.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
check that there is not any reference to Project3, so it doesn't appear on the solution

Reply:
just by adding the project into the solution